### PR TITLE
Add a link to the current release SHA

### PR DESCRIPTION
**Pull request by rgarner**
- Adds a link to the current `HEAD` commit on Github in a minimal footer
- Dependent on github links
- Dependent on presence of `HEROKU_SLUG_COMMIT` if in production
- Will show 'Release: unavailable' for any environment without
  `HEROKU_SLUG_COMMIT` or a git repo with a `HEAD`

We can do better than this, but this at least gives PM and DM an idea
of what's live. We can get to reasonable release naming and a changelog
next – this ticket is "minimal what's live"

screen shot 2017-08-10 at 12 10 53

### CODE REVIEW

**Comment by ztolley** [comment]
Could this be shown only if being run in development mode or under development environment and not shown for production?

Also, could it not be in the head but maybe a 'stuck' to the bottom right of the screen, or top right?

**Comment by rgarner** [comment]
Any of those things are possible, but the development thing is probably the least useful use case. The real use case here is for @sl33p and Veronique (username unknown) knowing what's on each 'production-like' (ie stage and prod) environment (though me buggering about with Heroku yesterday was a really useful use of a morning). This is a minimal starter for ten.

Another alternative is that we provide a totally separate URL (something like `/version` or `/changelog`) so that the real benefit of knowing what functional things are live is better realised.

In other words, this will come out at some point – is the benefit to be had from fettling it now primarily so that the header looks a bit less technical in show & tells?

**Comment by rgarner** [comment]
(was thinking of chucking something like https://github.com/devtyr/trello-releasenotes at our "Awaiting Deploy" column)

**Comment by ztolley** [comment]
If we use PR branches, the feature is in the url

**Comment by rgarner** [comment]
Didn't know that, and that's helpful, and this PR is about what's in stage/prod when that feature URL isn't there

**Comment by rgarner** [comment]
Lemme try the bottom right...
